dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -239,29 +272,43 @@ public  File backupIndex(String index, File toFile) throws IOException {
 			bw = new BufferedWriter(
 			        new OutputStreamWriter(zipOut), 500000); // 500K buffer
 
-			final String type=index.startsWith("sitesearch_") ? SiteSearchAPI.ES_SITE_SEARCH_MAPPING : "content";
-	        final String mapping = mappingAPI.getMapping(index, type);
+			final String type=index.startsWith("sitesearch_") ? SiteSearchAPI.ES_SITE_SEARCH_MAPPING
+					: "content";
+	        final String mapping = mappingAPI.getMapping(index);
 	        bw.write(MAPPING_MARKER);
 	        bw.write(mapping);
 	        bw.newLine();
 
-	        // setting up the search for all content
-            SearchResponse scrollResp =
-                client
-                    .prepareSearch(index)
-                    .setQuery(QueryBuilders.matchAllQuery())
-                    .setSize(100)
-                    .setScroll(TimeValue.timeValueMinutes(2))
-                    //_doc has no real use-case besides being the most efficient sort order.
-                    .addSort("_doc", SortOrder.DESC)
-                    .execute()
-                    .actionGet(INDEX_OPERATIONS_TIMEOUT_IN_MS);
+			SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+			searchSourceBuilder.query(QueryBuilders.matchAllQuery());
+			searchSourceBuilder.size(100);
+			searchSourceBuilder.timeout(TimeValue.timeValueMillis(INDEX_OPERATIONS_TIMEOUT_IN_MS));
+			//_doc has no real use-case besides being the most efficient sort order.
+			searchSourceBuilder.sort("_doc", SortOrder.DESC);
+
+
+			SearchRequest searchRequest = new SearchRequest();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'searchRequest' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)